server/view.go

```diff
@@ -95,6 +101,138 @@ func serveView(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func serveBatView(w http.ResponseWriter, r *http.Request) {
+	err := r.ParseForm()
+
+	if err != nil {
+		w.WriteHeader(http.StatusBadRequest)
+		w.Write([]byte(`{"error":"invalid URL query"}`))
+		return
+	}
+
+	GameID, err := strconv.Atoi(r.Form.Get("id"))
```

Comment:
As mentioned in another comment, this needs to be changed to contain platform and game ID as game IDs are not guaranteed to be unique across regions.